src/App.js

```diff
@@ -533,14 +535,24 @@ class App extends Component {
     const menuIconSelectedStyle = {
       fill: this.state.theme.menuIconSelected
     };
+    const appStyle = {
+      background: this.state.theme.appBackground,
+      color: this.state.theme.appText
```

Comment:
Maybe add fill here to give the proper color to the icons that are shown in the welcome text on empty dashboards.